docs/reference/search/async-search.asciidoc

```diff
@@ -98,7 +98,11 @@ some shards returned their results, or when the node that is coordinating the
 
 It is possible to block and wait until the search is completed up to a certain
 timeout by providing the `wait_for_completion_timeout` parameter, which
-defaults to `1` second.
+defaults to `1` second. When the async search completes within such timeout,
+the response won't include the ID as the results are not stored in the cluster.
```

Comment:
nit: s/the ID/the `id`/

Reply:
I think this is how @jrodewig prefers us to mention the id, I will defer to him :)